### PR TITLE
UML-1051 Translation tags added for few more templates

### PR DESCRIPTION
**Pull request by SeemaMenon**
# Purpose

https://opgtransform.atlassian.net/browse/UML-1051

Fixes UML-1051

## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/app/src/Actor/templates/actor/lpa-create-viewercode.html.twig [hunk 1]
```diff
@@ -35,15 +35,15 @@
...
                 <div class="govuk-grid-column-two-thirds">
                     {% if lpa.caseSubtype|lower == 'hw' %}
-                        <p class="govuk-body">The organisation could be a hospital, care home, or other health care provider.</p>
+                        <p class="govuk-body">{% trans %}The organisation could be a hospital, care home, or other health care provider.{% endtrans %}</p>
                     {% elseif lpa.caseSubtype|lower == 'pfa' %}
-                        <p class="govuk-body">The organisation could be a bank, energy provider, or another business.</p>
+                        <p class="govuk-body">{% trans %}The organisation could be a bank, energy provider, or another business.{% endtrans %}</p>
                     {% endif %}
 
                     {{ govuk_form_element(form.get('org_name'), { 'label': 'Organisation name', 'hint': 'This is for your reference only', 'attr' : {'class': 'govuk-!-width-one-half'} }) }}
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/lpa-create-viewercode.html.twig:49]: Hint and labels?

Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/lpa-create-viewercode.html.twig:49]: Covered by Gemma as part of UML-1000

Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/lpa-create-viewercode.html.twig:49]: Sorry my mistake, only validation messages updated as part of 1000. I shall add for the labels and hints now.

Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/lpa-create-viewercode.html.twig]: Added for label and hint

**Review by cooperaj — APPROVED** [review]
👍 Looking great

**Review by Raffers — APPROVED** [review]

**Pull request merged**